src/logmessage.cpp

```diff
@@ -27,35 +27,35 @@ namespace g3 {
    // helper for setting the normal log details in an entry
    std::string LogDetailsToString(const LogMessage& msg) {
       std::string out;
-      out.append("\n" + msg.timestamp() + "\t"
-                 + msg.level() + " [" + msg.file() + "->" + msg.function() + ":" + msg.line() + "]\t");
```

Comment:
I don't see how this will affect many. The fix is only meant to help people that mix g3log with a cout sink and std::cout. Removal of quotation does not bother me at least.

Looking at the txt files, the only visual thing that we lose is the alignment of the messages. In the original look they are mostly aligned which is nice.